agent-info/src/schema.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright 2024-Present Datadog, Inc. https://www.datadoghq.com/
+// SPDX-License-Identifier: Apache-2.0
+//! This module provides struct representing the info endpoint response
+use serde::Deserialize;
+use std::collections::HashMap;
+
+/// Wrapper for an agent info response storing the state hash from the agent
+#[derive(Clone, Deserialize, Default, Debug, PartialEq)]
+pub struct AgentInfo {
+    /// Hash of the info
+    pub state_hash: String,
+    /// Info response from the agent
+    pub info: AgentInfoStruct,
+}
+
+/// Schema of an agent info response
+#[allow(missing_docs)]
+#[derive(Clone, Deserialize, Default, Debug, PartialEq)]
+pub struct AgentInfoStruct {
```

Comment:
This should be updated with the latest changes to the RFC and here:
https://github.com/DataDog/datadog-agent/pull/28861

Reply:
done